src/test/kotlin/com/forgerock/sapi/gateway/ob/uk/tests/functional/payment/domestic/vrp/api/v3_1_10/CreateDomesticVrp.kt

```diff
@@ -303,8 +303,8 @@ class CreateDomesticVrp(val version: OBVersion, val tppResource: CreateTppCallba
         }
 
         // Then
-        assertThat((exception.cause as FuelError).response.statusCode).isEqualTo(401)
-        assertThat((exception.cause as FuelError).response.responseMessage).isEqualTo(com.forgerock.sapi.gateway.ob.uk.framework.errors.UNAUTHORIZED)
+        assertThat((exception.cause as FuelError).response.statusCode).isEqualTo(400)
+        assertThat((exception.cause as FuelError).response.responseMessage).isEqualTo(com.forgerock.sapi.gateway.ob.uk.framework.errors.BAD_REQUEST)
```

Comment:
This looks like the correct error according to the specifications found [here](https://openbankinguk.github.io/read-write-api-site3/v3.1.10/profiles/read-write-data-api-profile.html#incomplete-or-malformed-request-payload)

Reply:
I know that according to the OB spec, in this case, `"Token has incorrect scope or a security policy was violated"` we should get a `403 Forbidden`, but we also have the validations (initiation & risk sections) in RS which require a `400 Bad request` also according to the OB spec. 

At the moment with the `400 Bad request` case all the functional tests on the `Domestic VRPs` API seem to pass because we have already implemented the initiation validation service for the `Domestic VRPs` (request vs. consent) in RS and it is expecting a `400 Bad request`. 
See PR: https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rs/pull/120 

This is the `OBRIErrorType` we have in that case: `REQUEST_VRP_INITIATION_DOESNT_MATCH_CONSENT(HttpStatus.BAD_REQUEST, OBStandardErrorCodes1.UK_OBIE_RESOURCE_CONSENT_MISMATCH, "The provided initiation data differs from that in the matching consent"`

So we basically have: https://openbankinguk.github.io/read-write-api-site3/v3.1.10/profiles/read-write-data-api-profile.html#incomplete-or-malformed-request-payload vs. https://openbankinguk.github.io/read-write-api-site3/v3.1.10/resources-and-data-models/pisp/domestic-payments.html#notes

Reply:
Sorry for any confusion here. 

My point was that, because it is the contents of the request that is invalid, then 400 BAD_REQUEST is correct, and the functional tests were testing for the wrong error previously! 

So I think this PR is good. I'm a little concerned that the assert was checking for 401 (UNAUTHORIZED) because that was not what the spec says the result should be, so it may be a case of writing a test to basically pass based on the API behaviour, rather than being written against what the Specifications said!